test/e2e/e2e-test/src/test/java/org/apache/skywalking/e2e/kafka/KafkaE2E.java

```diff
@@ -200,8 +202,9 @@ void so11y() throws Exception {
                 for (String metricsName : ALL_SO11Y_LABELED_METRICS) {
                     LOGGER.info("verifying service instance response time: {}", instance);
                     final List<ReadMetrics> instanceMetrics = graphql.readLabeledMetrics(
-                        new ReadMetricsQuery().stepByMinute().metricsName(metricsName)
+                        new ReadLabeledMetricsQuery().stepByMinute().metricsName(metricsName)
                                               .serviceName(service.getLabel()).instanceName(instance.getLabel())
+                                              .labels(Arrays.asList("50", "70", "90", "99"))
```

Comment:
What is this? A previous bug?

Reply:
The previous version of the labeled metrics query is not supported `label` customize. So I just add a new query type and add the `labels` property to customize in the `gql` file.